Scripts/Runtime/VoiceService.cs

```diff
@@ -48,6 +55,13 @@ public VoiceEvents VoiceEvents
         /// </summary>
         /// <param name="transcription"></param>
         public abstract void Activate(string transcription);
+        
+        /// <summary>
+        /// Send text data for NLU processing with custom request options.
+        /// </summary>
+        /// <param name="transcription"></param>
+        /// <param name="requestOptions"></param>
+        public abstract void Activate(string transcription, WitRequestOptions requestOptions);
```

Comment:
s/transcription/text

Reply:
I was going off of the established parameter name, which was previously `transcription`. But this is a good change, I'll fix all of these instances.